Neos.Neos/Resources/Private/Fusion/Prototypes/NodeLink.fusion

```diff
@@ -0,0 +1,34 @@
+# Renders an <a> tag based on Neos.Neos:NodeUri pointing to a node
+#
+prototype(Neos.Neos:NodeLink) < prototype(Neos.Fusion:Tag) {
+  node = null
+  additionalParams = Neos.Fusion:RawArray
+  arguments = Neos.Fusion:RawArray
+  argumentsToBeExcludedFromQueryString = Neos.Fusion:RawArray
+  addQueryString = false
+  absolute = false
+  baseNodeName = 'documentNode'
+
+  @context.node = ${this.node}
+  @context.additionalParams = ${this.additionalParams}
+  @context.arguments = ${this.arguments}
+  @context.argumentsToBeExcludedFromQueryString = ${this.argumentsToBeExcludedFromQueryString}
+  @context.addQueryString = ${this.addQueryString}
+  @context.absolute = ${this.absolute}
+  @context.baseNodeName = ${this.baseNodeName}
+
+  tagName = 'a'
+  attributes {
+    href = Neos.Neos:NodeUri {
+      node = ${node}
+      additionalParams = ${additionalParams}
+      arguments = ${arguments}
+      argumentsToBeExcludedFromQueryString = ${argumentsToBeExcludedFromQueryString}
+      addQueryString = ${addQueryString}
+      absolute = ${absolute}
+      baseNodeName = ${baseNodeName}
+    }
+  }
+
+  content = ${q(this.node).property('_label')}
```

Comment:
```suggestion
  content = ${this.node.label}
```
like so?

Reply:
Is adjusted to ``node.label`` as ``this.node`` was put into ``@context.node`` anyways.